sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/delete_key_operation.hpp

```diff
@@ -77,6 +77,13 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
         Azure::Core::Response<Azure::Security::KeyVault::Keys::DeletedKey> response);
 
   public:
+    /**
+     * @brief Get the #Azure::Core::Http::RawResponse of the operation request.
+     * @return A pointer to #Azure::Core::Http::RawResponse or null if the RawResponse does not
+     * exist.
+     */
```

Comment:
Add `@note Does not give up ownership of the RawResponse.`

Reply:
I can call this method, and update the RawResponse :).  It might mess up the internal state of the Operation<T>.
If we don't want to return a `const&` or a `RawResponse const*`, then we should return a a copy of the entire RawResponse.